cmd/flightctl-periodic/main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"github.com/flightctl/flightctl/internal/config"
+	"github.com/flightctl/flightctl/internal/periodic_checker"
+	"github.com/flightctl/flightctl/internal/store"
+	"github.com/flightctl/flightctl/pkg/log"
+	"github.com/sirupsen/logrus"
+)
+
+func main() {
+	log := log.InitLogs()
+	log.Println("Starting periodic")
+
+	cfg, err := config.LoadOrGenerate(config.ConfigFile())
+	if err != nil {
+		log.Fatalf("reading configuration: %v", err)
+	}
+	log.Printf("Using config: %s", cfg)
+
+	logLvl, err := logrus.ParseLevel(cfg.Service.LogLevel)
+	if err != nil {
+		logLvl = logrus.InfoLevel
+	}
+	log.SetLevel(logLvl)
+
+	log.Println("Initializing data store")
+	db, err := store.InitDB(cfg, log)
+	if err != nil {
+		log.Fatalf("initializing data store: %v", err)
+	}
+
+	store := store.NewStore(db, log.WithField("pkg", "store"))
+	defer store.Close()
+
+	if err := store.InitialMigration(); err != nil {
```

Comment:
```suggestion
	if err := store.InitialMigration(); err != nil {
```

See my comments above.

Reply:
Good point.  Options are:
1. Use kubernetes leader election on one of the components (for example api).  The downside is that it will not work for podman since the solution is bound to kubernetes.
2. Use a database mechanism to lock - might need extra table.

Reply:
Added code in **store** package to handle single invocation of auto-migration.